packages/podge/podge.0.2/opam

```diff
@@ -0,0 +1,31 @@
+opam-version: "1.2"
+maintainer: "Edgar Aroutiounian <edgar.factorial@gmail.com>"
+authors: [ "Edgar Aroutiounian <edgar.factorial@gmail.com>" ]
+license: "GPL-3.0"
```

Comment:
one thing to bear in mind is that the GPL is not a good license for an OCaml library. If you want to maintain copyleft, then the LGPL+OCaml linking exception works much better due the static linking that the OCaml compiler does.